rest/config.go

```diff
@@ -87,6 +87,7 @@ type ServerConfig struct {
 	SkipRunmodeValidation          bool                     `json:"skip_runmode_validation,omitempty"` // If this is true, skips any config validation regarding accel vs normal mode
 	Unsupported                    *UnsupportedServerConfig `json:"unsupported,omitempty"`             // Config for unsupported features
 	RunMode                        SyncGatewayRunMode       `json:"runmode,omitempty"`                 // Whether this is an SG reader or an SG Accelerator
+	ReplicatorCompression          *int                     `json:"replicator_compression,omitempty"`  // BLIP data compression level (0-9)
```

Comment:
Can you add the default compression level to the comment?

Reply:
@jamiltz what's the best process for adding docs for new params?  Github PR or Jira issue?

Reply:
GitHub PR, here's one with a placeholder https://github.com/couchbaselabs/couchbase-mobile-portal/pull/837

Reply:
I could, but it would tend to become obsolete if we ever change the default level in go-blip, especially since that's in a separate repo...